app/controllers/account/transactions_controller.rb

```diff
@@ -54,4 +56,14 @@ def entry_params
               end
             end
     end
+
+    def partial_options
+      params
+        .require(:partial_options)
+        .permit(:selectable)
```

Comment:
I wasn’t sure if it was worth adding all the other locals that the transaction partial receives (editable, short, show_tags, etc.).